client/app/scripts/charts/node.js

```diff
@@ -83,6 +83,9 @@ export default class Node extends React.Component {
     if (this.props.pseudo) {
       classNames.push('pseudo');
     }
+    if (this.props.shape === 'cloud') {
+      classNames.push('cloud');
+    }
```

Comment:
This is a good one:

* clouds have strokes set on their attrs via JS because of dynamic scaling
* all shapes have strokes set their width via CSS
* CSS overrides widths set on attrs -> cloud shapes get wrong (from CSS) widths

my fix for this introduces a `.cloud` selector that can be used to address all nodes except the cloud to set widths via CSS.